src/Pages/AboutUs/AboutUs.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react';
+import './AboutUs.css';
+import AboutUsCard from './AboutUsCard.js';
+import AboutUsCircle from './AboutUsCircle.js';
+import {getAllOfficers} from '../../APIFunctions/OfficerManager';
+import Header from '../../Components/Header/Header';
+import Footer from '../../Components/Footer/Footer';
+class AboutUs extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      officers: [],
+      headerProps: {
+        title: 'About Us'
```

Comment:
Why is the header props part of the state? Are changing it some way. If you want to have a different header for the About Us and About Us Manager you can just pass it as a regular prop and add a header attribute in the constructor